upstream-community-operators/etcd/etcd.package.yaml

```diff
@@ -1,5 +1,5 @@
-#! package-manifest: deploy/chart/catalog_resources/rh-operators/etcdoperator.v0.9.2.clusterserviceversion.yaml
-packageName: etcd
+#! package-manifest: deploy/chart/catalog_resources/rh-operators/etcdoperator-community.v0.9.2.clusterserviceversion.yaml
+packageName: etcd-community
```

Comment:
Upstream operators don't need this suffix, since we do not expect to have both upstream and community operators sources.